test/DropdownSpec.js

```diff
@@ -416,7 +416,7 @@ describe('Dropdown', () => {
     // I am fairly confident that the failure is due to a test specific conflict and not an actual bug.
     it('when open and the key "esc" is pressed the menu is closed and focus is returned to the button', () => {
       const instance = React.render(
-        <Dropdown defaultOpen alwaysFocusNextOnOpen id='test-id'>
+        <Dropdown defaultOpen role="menuitem" id="test-id">
```

Comment:
Yes, because of https://github.com/react-bootstrap/react-bootstrap/pull/1375/files#diff-7d1cbca6e042a330878fd99a6c9a90b9R427. I need this to make the test pass.